other-topics/retrieve-componenets-files/scripts/get_raw_files

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+# Directories.
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+OUTPUT_DIR=${OUTPUT_DIR:-${SOURCE_DIR}/_out}
+
+CAPI_CONFIG="/home/${USER}/go/src/sigs.k8s.io/cluster-api/config"
+CAPBM_CONFIG="/home/${USER}/go/src/github.com/metal3-io/cluster-api-provider-baremetal/config"
+BMO_CONFIG="/home/${USER}/go/src/github.com/metal3-io/baremetal-operator/deploy"
+
+# Binaries
+ENVSUBST=${ENVSUBST:-envsubst}
+command -v "${ENVSUBST}" >/dev/null 2>&1 || echo -v "Cannot find ${ENVSUBST} in path."
+
+mkdir -p "${OUTPUT_DIR}/capi"
+mkdir -p "${OUTPUT_DIR}/bmo"
+mkdir -p "${OUTPUT_DIR}/capbm"
+
+# Generate cluster resources.
```

Comment:
remove this as well.

Reply:
This comment refers to the removal of the entire file